junwoo/app.js

```diff
@@ -29,7 +30,38 @@ app.get("/ping", cors(), (req, res) => {
   res.json({ message: "pong" });
 });
 
-const PORT = process.env.PORT;
+app.post("/users/signup", async (req, res) => {
+  const { name, email, age } = req.body;
+
+  const result = await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      age
+      ) VALUES (?,?,?);
+      `,
+    [name, email, age]
+  );
+  console.log(result);
+  res.status(201).json({ message: "userCreated" });
+});
+
+app.post("/posts/create", async (req, res) => {
```

Comment:
추후 RESTful API에 대한 세션을 접할 예정입니다.
현재도 RESTful API를 작성을 연습해보는 것이 좋습니다.

우선, 엔드포인트에서 동사를 사용하지 않습니다. 특정 리소스에 대한 값만으로도 우리는 해당 엔드포인트가 어떤 API를 가리키고 어떤 값을 반환할 지 추측할 수 있습니다.

GET Method -> `app.get('/posts', ...)` ( o )
POST Method -> `app.post('/add/posts', ...)` ( x ) // `app.post('/posts', ...)` ( o )

```suggestion
app.post("/posts", async (req, res) => {
```